src/EditorFeatures/Test/CodeGeneration/CodeGenerationTests.cs

```diff
@@ -262,7 +262,7 @@ public partial class CodeGenerationTests
                 exception = e;
             }
 
-            var expectedMessage = string.Format(WorkspacesResources.Cannot_generate_code_for_unsupported_operator_0, method.Name);
+            var expectedMessage = string.Format(WorkspaceExtensionsResources.Cannot_generate_code_for_unsupported_operator_0, method.Name);
```

Comment:
a bunch of resources had to move as well.  that's the bulk of the add/removes in this PR.